package.json

```diff
@@ -21,15 +21,19 @@
   ],
   "contributors": [],
   "devDependencies": {
-    "eslint": "^7.14.0"
+    "eslint": "^7.14.0",
+    "eslint-plugin-node": "^11.1.0"
```

Comment:
this should be part of dependencies, otherwise it won't be included in the including project